bedrock/firefox/templates/firefox/new/batm/machine-a.html

```diff
@@ -0,0 +1,73 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-new-batm-anim', ['en-US']) %}
+    {% javascript 'experiment_firefox_new_batm_anim' %}
```

Comment:
Hmm, I don't think this traffic cop experiment will work the way this is set up. You need to trigger the experiment from a page that isn't already one of the variations. Traffic cop won't do this in order to [avoid potential infinite redirects](https://github.com/mozilla/bedrock/blob/master/media/js/base/mozilla-traffic-cop.js#L98-L100). Is that right, @jpetto?

I'm a little confused why they want to do an experiment like this anyway. Can't they already control which variations they are sending traffic to directly from the ads, and then adjust as they monitor performance?